src/services/polkit/identity.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <qobject.h>
+#include <qqmlintegration.h>
+
+typedef struct _PolkitIdentity PolkitIdentity;
+
+namespace qs::service::polkit {
+//! Represents a user or group that can be used to authenticate.
+class Identity: public QObject {
+	Q_OBJECT;
+
+	// clang-format off
+	/// The Id of the identity. If the identity is a user, this is the user's uid. See @@isGroup.
+	Q_PROPERTY(id_t id READ id CONSTANT);
```

Comment:
Please cast/rename to a qt primitive type in the interface. This will show up as `unknown` in the doc generator.